pkgs/development/tools/rye/default.nix

```diff
@@ -56,6 +56,7 @@ rustPlatform.buildRustPackage rec {
   meta = with lib; {
     description = "A tool to easily manage python dependencies and environments";
     homepage = "https://github.com/mitsuhiko/rye";
+    changelog = "https://github.com/mitsuhiko/rye/releases/tag/${version}";
```

Comment:
```suggestion
    changelog = "https://github.com/mitsuhiko/rye/blob/${versopm}/CHANGELOG.md";
```

Reply:
I'll leave it upto you to decide which URL is better, this is just a suggestion

Reply:
Thank you for the suggestion.
I think that the GH release page is better because it has more info (mention of new contributors + possibility to check the diff).
Hence I propose to keep it this way.